playwright/playwright.config.ts

```diff
@@ -0,0 +1,71 @@
+import {resolve} from 'path';
+
+import type {PlaywrightTestConfig} from '@playwright/experimental-ct-react';
+import {defineConfig, devices} from '@playwright/experimental-ct-react';
+import react from '@vitejs/plugin-react';
+
+function pathFromRoot(p: string) {
+    return resolve(__dirname, '../', p);
+}
+
+const reporter: PlaywrightTestConfig['reporter'] = [];
+
+reporter.push(
+    ['list'],
+    [
+        'html',
+        {
+            open: process.env.CI ? 'never' : 'on-failure',
+            outputFolder: resolve(process.cwd(), 'playwright-report'),
+        },
+    ],
+);
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+    testDir: pathFromRoot('src'),
+    testMatch: '*/__tests__/*.visual.test.tsx',
+    updateSnapshots: process.env.UPDATE_REQUEST ? 'all' : 'missing',
+    snapshotPathTemplate:
+        '{testDir}/{testFileDir}/{testFileName}-snapshots/{arg}{-projectName}-linux{ext}',
```

Comment:
Let's do this in the Jest manner:

```suggestion
        '{testDir}/{testFileDir}/__snapshots__/{arg}{-projectName}-linux{ext}',
```